### PR TITLE
pybind/cephfs: add cephfs python API removexattr()

### PR DESCRIPTION
**Pull request by zhangbingyi1**
Signed-off-by: bingyi zhang <zhangbingyi@umcloud.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: src/pybind/cephfs/cephfs.pyx [hunk 1]
```diff
@@ -1096,6 +1097,26 @@ cdef class LibCephFS(object):
         if ret < 0:
             raise make_ex(ret, "error in setxattr")
 
+    def removexattr(self, path, name):
```
Comment (tchaikov) [inline src/pybind/cephfs/cephfs.pyx]: mind adding a test in `src/test/pybind/test_cephfs.py`?

Comment (zhangbingyi1) [reply src/pybind/cephfs/cephfs.pyx]: I've added the test, please check it.

**Comment by tchaikov** [comment]
could you revise the "Signed-off-by" line in commit message, it reads

> Signed-off-by: bingyi zhang zhangbingyi@umcloud.com

it should be

> Signed-off-by: bingyi zhang <zhangbingyi@umcloud.com>

also, please squash these two commits into a single one.

**Comment by zhangbingyi1** [comment]
@tchaikov ok, please check it

File: src/pybind/cephfs/cephfs.pyx [hunk 2]
```diff
@@ -1096,6 +1097,26 @@ cdef class LibCephFS(object):
...
+        """
+        Remove an extended attribute on a file.
```
Comment (varshar16) [inline src/pybind/cephfs/cephfs.pyx]: Few small changes
```suggestion
        Remove an extended attribute of a file.
```

File: src/pybind/cephfs/cephfs.pyx [hunk 3]
```diff
@@ -1096,6 +1097,26 @@ cdef class LibCephFS(object):
...
+
+        :param path: the path to the file.
+        :param name: the name of the extended attribute to set.
```
Comment (varshar16) [inline src/pybind/cephfs/cephfs.pyx]:
```suggestion
        :param name: name of the extended attribute to remove.
```

**Comment by tchaikov** [comment]
```
140 - unittest_chain_xattr (Timeout)
```
jenkins test make check

**Comment by zhangbingyi1** [comment]
@varshar16  I have changed the code comment. Please check it.

**Comment by varshar16** [comment]
jenkins test make check arm64

**Pull request merged**